exceptions.cabal

```
@@ -45,8 +45,10 @@ library
     base                       >= 4.9      && < 5,
     mtl                        >= 2.2      && < 2.4,
     stm                        >= 2.2      && < 3,
-    template-haskell           >= 2.11     && < 2.25,
     transformers               >= 0.5.2.0  && < 0.7
+  if impl(ghc)
+    build-depends:
+      template-haskell         >= 2.11     && < 2.25,
```

Remove the comma at the end here to make it parse with `cabal` (see the CI failures):

```suggestion
      template-haskell         >= 2.11     && < 2.25
```